vhdbuilder/packer/backfill-cleanup.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash -x
+
+EXPIRATION_IN_HOURS=168
+# convert to seconds so we can compare it against the "tags.now" property in the resource group metadata
+(( expirationInSecs = ${EXPIRATION_IN_HOURS} * 60 * 60 ))
+# deadline = the "date +%s" representation of the oldest age we're willing to keep
+(( deadline=$(date +%s)-${expirationInSecs%.*} ))
+
+if [[ "${MODE}" != "windowsVhdMode" ]]; then
+  exit 0
+fi
+
+# any build not in dry-run mode for Windows must be a check-in build
+if [[ "${BACKFILL_RESOURCE_DELETION}" == "False" ]] && [[ "${DRY_RUN}" == "True" ]]; then
```

Comment:
This can be more simple to remove the parameter DRY_RUN.
 In check-in pipeline, the default value of  `BACKFILL_RESOURCE_DELETION` is `false` and we can set it to `true` manually. It works without `DRY_RUN`.
We can always set `BACKFILL_RESOURCE_DELETION` to `true` in prod pipeline instead of using `DRY_RUN`.